### PR TITLE
[docs] Make module names consistent in yaml files

### LINKED ISSUE #10697: [Metricbeat] Cleanup module names in docs
We are not consistent on the naming of modules in the docs. We have `aws` and `couchdb` lower case, but most others start with an upper case letter. And we have `HTTP` but `Nats`, should it be `NATS`? https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-modules.html

All module names / titles must be checked for the correct writing according to the naming of the actual service. The above are just examples.

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #10697  (sort of)

**What this PR does:**
* Capitalizes module names to make them consistent across the Metricbeat docs. This is a quick fix to resolve inconsistent capitalization in the TOC.

**What this PR does not do:**
* Does not fix instances where we use third-party product names incorrectly. I've opened an issue in the style-guide repo to get clear guidance so we can make recommendations to the dev team. 
* Does not change module names that are hard coded in the docs. We can clean this up later, if necessary.
* Ignores the recommendation in elastic/beats#10697 about wrapping the lowercase module name in backtics. We need to revisit this discussion as part of the doc design for the integration packages.

IMO, we should make the module docs as clean as possible for the migration, but not waste time cleaning up inconsistencies that don't matter to the user (unless the inconsistencies make the docs look sloppy and unprofessional).

### CODE REVIEW

File: metricbeat/docs/modules/aws/sqs.asciidoc [hunk 1]
```diff
@@ -3,7 +3,7 @@ This file is generated! See scripts/mage/docs_collector.go
 ////
 
 [[metricbeat-metricset-aws-sqs]]
-=== aws sqs metricset
+=== Aws sqs metricset
```
Comment (ruflin) [inline metricbeat/docs/modules/aws/sqs.asciidoc:6]: AWS? Applies also to the others?

SQS? @kaiyan-sheng

File: x-pack/metricbeat/module/aws/_meta/fields.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - key: aws
-  title: "aws"
+  title: "Aws"
```
Comment (exekias) [inline x-pack/metricbeat/module/aws/_meta/fields.yml:2]: This should probably be AWS cc @kaiyan-sheng

File: x-pack/metricbeat/module/activemq/_meta/fields.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 - key: activemq
-  title: "activemq"
+  title: "Activemq"
```
Comment (exekias) [inline x-pack/metricbeat/module/activemq/_meta/fields.yml:2]:
```suggestion
  title: "ActiveMQ"
```

File: metricbeat/docs/modules/aws/sqs.asciidoc (hunk 1 above)
Comment (dedemorton) [reply metricbeat/docs/modules/aws/sqs.asciidoc:6]: Changing AWS, but let's keep the metricset names lowercase for now.

**Comment by dedemorton** [comment]
I've fixed the issues noticed during the review plus found a couple other places where we are inconsistent.

This doesn't fix everything, but at least it looks like we know how to capitalize things consistently now. :-)

**Comment by dedemorton** [comment]
Note that I've created https://github.com/elastic/beats/issues/18220 to track the additional cleanup work.

**Pull request merged**